presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveQlToPrestoTranslator.java

```diff
@@ -0,0 +1,242 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Iterators;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import com.google.common.collect.Streams;
+import io.prestosql.sql.parser.ParsingOptions;
+import io.prestosql.sql.parser.SqlParser;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static io.prestosql.plugin.hive.HiveQlToPrestoTranslator.translateHiveQuotesToPresto;
+import static io.prestosql.testing.assertions.Assert.assertEquals;
+import static java.lang.String.format;
+import static java.util.Collections.nCopies;
+
+public class TestHiveQlToPrestoTranslator
```

Comment:
Don't rename the class. This is preventing Git from showing a proper diff for things like `testStringLiterals` which did not change.

Reply:
Oops; fixed.